modules/react-components/src/components/input/search-with-filter-labels.tsx

```diff
@@ -154,7 +154,27 @@ export const SearchWithFilterLabels: FunctionComponent<PropsWithChildren<SearchW
                     )
             }
             {
-                (filterLabels && Array.isArray(filterLabels) && filterLabels.length > 0) && (
+                isLoading ? (
+                    <Card.Group style={ { borderRadius: "50px" } }>
+                        {
+                            [ ...Array(4) ].map(() => {
+                                return (
+                                    <>
+                                        <Card style={ { borderRadius: "50px", boxShadow: "none",  width: "65px" } }>
```

Comment:
I think the use of inline styles is not a good practice and it will lead to inconsistencies. Can't we create a class and override styles of the related element for these inline styles?

Reply:
This is added as a TODO. Will be addressing this in another PR.